contrib/nydus-test/functional-test/test_private.py

```diff
@@ -0,0 +1,44 @@
+from conftest import ANCHOR
+import posixpath
+import pytest
+from nydusify import Nydusify
+from rafs import RafsMount, RafsConf, Backend
+from nydus_anchor import NydusAnchor
+from workload_gen import WorkloadGen
+
+from utils import read_images_array
+
+REGISTRY_ADDR = "https://docker.alibaba-inc.com/#/imageList"
```

Comment:
This test file aims at testing internal business system container images. Besides a list of private/internal images, it is quite similar to other test cases. So I think we might drop the test file.

Reply:
OK, thanks a lot! ; )